### DISCUSSION TITLE
How do I connect to the Oxford Instruments MercuryiPS using USB instead of the Internet network port?

### DISCUSSION THREAD

**Discussion by zaizhe** [microsoft/Qcodes#4418] [Q&A]

**Comment by jenshnielsen** [comment]
@zaizhe Thanks for the question. I took the liberty of converting this from an issue to the discussion which is better suited for questions. 
I have not tested this my self but we have a VISA version of the driver here https://qcodes.github.io/Qcodes/api/generated/qcodes.instrument_drivers.oxford.html#qcodes.instrument_drivers.oxford.MercuryiPS_VISA.MercuryiPS

Which you should be able to connect using multiple interfaces. PyVISA has this handy guide that shows you how to connect using different interfaces.
